lib/ansible/utils/path.py

```diff
@@ -103,3 +104,9 @@ def basedir(source):
         dname = os.path.abspath(dname)
 
     return to_text(dname, errors='surrogate_or_strict')
+
+
+def cleanup_tmp_dir(path):
```

Comment:
I'm not sure if I put this in the best place. Putting it in `bin/ansible` meant I couldn't import it.

Reply:
I agree, it should not be in `bin/ansible` but some other location, importable throughout the codebase.

Reply:
agreed, ignore my previous comment